### PR TITLE
Card view

### PR DESCRIPTION
**Pull request by BQuinteros**
added a card view in the fragment dialog to show a hero
Screenshot_1577462547
Screenshot_1577462555

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -48,6 +49,7 @@ dependencies {
             generalTestDependencies.junit,
             generalTestDependencies.mockitoInline,
             generalTestDependencies.kotlinTest
+    implementation 'com.android.support:cardview-v7:26.1.0'
```
Comment (SartiMau) [inline app/build.gradle]: why do you have android.support:cardview and androidx.cardview, keep only one pls

File: app/src/main/res/layout/fragment_character.xml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    <android.support.constraint.ConstraintLayout
+        android:layout_height="match_parent"
+        android:layout_width="match_parent">
+
+        <ImageView
+            android:contentDescription="@string/image_description"
+            android:id="@+id/image_thumbnail"
+            android:layout_height="match_parent"
+            android:layout_margin="5dp"
```
Comment (SartiMau) [inline app/src/main/res/layout/fragment_character.xml]: use dimens pls

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_character.xml (hunk 2 above)
Comment (BQuinteros) [reply app/src/main/res/layout/fragment_character.xml]: done

File: app/build.gradle (hunk 1 above)
Comment (BQuinteros) [reply app/build.gradle]: done

**Review by SartiMau — APPROVED** [review]

**Review by oadussang — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**